packages/bns/src/index.ts

```diff
@@ -0,0 +1,748 @@
+import {
+  bufferCV,
+  callReadOnlyFunction,
+  ClarityType,
+  ClarityValue,
+  cvToString,
+  getAddressFromPrivateKey,
+  hash160,
+  makeRandomPrivKey,
+  makeUnsignedContractCall,
+  privateKeyToString,
+  ResponseErrorCV,
+  StacksTransaction,
+  standardPrincipalCV,
+  UnsignedContractCallOptions,
+} from '@stacks/transactions';
+
+import { StacksNetwork } from '@stacks/network';
+
+import { bufferCVFromString, decodeFQN, getZonefileHash, uintCVFromBN } from './utils';
+
+import BN from 'bn.js';
+
+export const BNS_CONTRACT_ADDRESS = 'ST000000000000000000002AMW42H';
```

Comment:
Should this have both testnet and mainnet addresses?
```ts
export const enum BnsContracAddress {
  mainnet = 'SP000000000000000000002Q6VF78',
  testnet = 'ST000000000000000000002AMW42H',
}
```

Reply:
ah yes! good catch